src/main/java/core/basesyntax/service/strategy/FruitStrategy.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.dto.FruitTransactionDto;
+import java.util.List;
+
+public class FruitStrategy {
+    private final List<OperationHandler> handlers;
+
+    public FruitStrategy(List<OperationHandler> handlers) {
+        this.handlers = handlers;
+    }
+
+    public List<OperationHandler> getHandlers(FruitTransactionDto dto) {
```

Comment:
Strategy's main point is to give you the correct handler depending on some value, so:
```suggestion
    public OperationHandler findHandlerFor(FruitTransactionDto dto) {
```
or
```suggestion
    public OperationHandler findHandlerFor(Operation operation) {
```